core/common/src/main/java/com/alipay/sofa/rpc/common/utils/CommonUtils.java

```diff
@@ -275,4 +272,43 @@ public static String join(Collection collection, String separator) {
         }
         return sb.length() > 0 ? sb.substring(0, sb.length() - separator.length()) : StringUtils.EMPTY;
     }
+
+    /**
+     * 将String数组转化为map
+     *
+     * @param pairs  目标数组
+     * @return HashMap
+     */
+    public static Map<String, String> toStringMap(String... pairs) {
```

Comment:
The method is just used in `ConsulURL`, can move to this method to `ConsulURLUtils` or  `ConsulURL`.